### DISCUSSION TITLE
[Help] - (Seeking customisable default zoom)

### DISCUSSION THREAD

**Discussion by SHHSSH** [piroor/treestyletab#3564] [Q&A]
How do you customise the default zoom for TST?

I require mine to be 2x Ctrl+Scroll Down for macros to work effectively.
Is there any CSS code perhaps I could use? Or a setting other than "Zoomable" toggling?

Thanks.

**Comment by piroor** [comment]
Try this code with TST options => Advanced => User Style Sheet:

```css
:root { zoom: 120%; }
```

**Reply by SHHSSH** [reply]
> Try this code with TST options => Advanced => User Style Sheet:
> 
> ```css
> :root { zoom: 120%; }
> ```

Unfortunately that doesn't work due to having it not be the same as the Ctrl+ScrollUp/Down on TST sidebar.
What function is happening for that affect within TST? As I'd like to perhaps mimic it by way of option setting, or css code please.

As I don't think zoom within that CSS code is functioning the same.
